library_generation/generate_repo.py

```diff
@@ -72,6 +75,10 @@ def generate(
     target_library_names: str,
     repository_path: str,
 ):
+    if generation_config_yaml is None:
```

Comment:
We can discuss this in https://github.com/googleapis/sdk-platform-java/pull/2615#discussion_r1583846508 
If we want to locally test subcomponents of the main script, this may be convenient.